sdk/storage/azure-storage-blob-changefeed/README.md

```diff
@@ -1,3 +1,6 @@
+## _Disclaimer_
```

Comment:
Could we add a note to the changelog for the next release for each of these packages?

Reply:
Good idea. Done.